### PR TITLE
#2625 Create sequence screen and load all triggers

### PR DESCRIPTION
**Pull request by ermin-muratovic**
Create sequence screen and load/display all triggers.

Since all triggers are actually "root" events, as shown in the service screen, the root events are polled regularly and distributed to the entities for the sequence and the service screen.

The `project-board` component will be reduced in to smaller components and is part of #1989 

The detail view in the `Sequence-Tab` is part of #2627 and the tiles will show more details which is part of #2628

image

### CODE REVIEW

File: bridge/client/app/_models/trace.ts [hunk 1]
```diff
@@ -195,16 +198,19 @@ class Trace {
       if(this.isProblem() && this.isProblemResolvedOrClosed()) {
         this.label = EVENT_LABELS[EventTypes.PROBLEM_RESOLVED];
       } else if(this.isApprovalFinished()) {
-        this.label = EVENT_LABELS[EventTypes.APPROVAL_FINISHED][this.data.approval?.result] || this.type;
+        this.label = EVENT_LABELS[EventTypes.APPROVAL_FINISHED][this.data.approval?.result] || this.type.split(".").slice(-2)[0];
       } else {
-        this.label = EVENT_LABELS[this.type] || this.type;
+        this.label = EVENT_LABELS[this.type] || this.type.split(".").slice(-2)[0];
```
Comment (dominikmessner) [inline bridge/client/app/_models/trace.ts:203]: [Suggestion] ```this.type.split(".").slice(-2)[0];``` could be extracted to a function so it's a bit easier to understand what it extracts (also in line 201)

**Review by dominikmessner — APPROVED** [review]

**Review by dominikmessner — APPROVED** [review]

**Review by dominikmessner — DISMISSED** [review]

File: bridge/client/app/app.module.ts [hunk 2]
```diff
@@ -36,10 +36,12 @@ import { DtTagModule } from '@dynatrace/barista-components/tag';
 import { DtTopBarNavigationModule } from "@dynatrace/barista-components/top-bar-navigation";
 import { DtCopyToClipboardModule } from "@dynatrace/barista-components/copy-to-clipboard";
 import { DtToggleButtonGroupModule } from "@dynatrace/barista-components/toggle-button-group";
+import {DtQuickFilterModule} from "@dynatrace/barista-components/experimental/quick-filter";
```
Comment (christian-kreuzberger-dtx) [inline bridge/client/app/app.module.ts:39]: please align formatting of this import with the other imports (Note to myself: we need a linter :D )

File: bridge/client/app/_models/trace.ts (hunk 1 above)
Comment (ermin-muratovic) [reply bridge/client/app/_models/trace.ts]: Thanks. Extracted it to a function and actually changed also the returned value.

**Comment by christian-kreuzberger-dtx** [comment]
Looks good, I'll give the final approval once the linter issue is fixed.

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

File: bridge/client/app/app.module.ts (hunk 2 above)
Comment (ermin-muratovic) [reply bridge/client/app/app.module.ts]: Yes, we should activate a linter tool. I create a follow-up issue for this, see #2915

Comment (dominikmessner) [reply bridge/client/app/app.module.ts]: Prettier is maybe also interesting and could make it a bit easier to introduce style rules to the existing code base (sorry, I'm a Prettier-fanboy :smile: )

**Pull request merged**